www-nginx/ngx-encrypted-session/ngx-encrypted-session-0.09.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="encrypted-session-nginx-module"
+inherit nginx-module
+
+DESCRIPTION="An NGINX module that encrypts and decrypts NGINX variable values"
+HOMEPAGE="https://github.com/openresty/encrypted-session-nginx-module"
+SRC_URI="
+	https://github.com/openresty/encrypted-session-nginx-module/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+"
+NGINX_MOD_S="${WORKDIR}/${MY_PN}-${PV}"
+
+LICENSE="BSD-2"
+SLOT="0"
+
+DEPEND="
+	dev-libs/openssl
+	www-nginx/ngx_devel_kit
+"
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	append-cflags -DNDK
```

Comment:
What is NDK? Should the eclass be setting it if everything needs it?

Reply:
Ah, OK. It should be fine as-is then (I wasn't sure if it was always needed, or very common).